src/filters/gaussianBlur.ts

```diff
@@ -87,6 +87,10 @@ export function gaussianBlur(
     return separableConvolution(image, kernelX, kernelY, {
       borderType,
     });
+  } else {
+    throw new Error(
+      'sigma is undefined. you must define sigma or sigmaX and sigmaY',
+    );
```

Comment:
This qualifies as a `TypeError`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypeError

```suggestion
    throw new TypeError(
      'you must either define sigma or sigmaX and sigmaY in the options argument',
    );
```